extensions/xcode/package.json

```diff
@@ -239,6 +239,21 @@
       "description": "Removes the ~/Library/Developer/Xcode/DerivedData directory.",
       "mode": "no-view"
     },
+    {
+      "name": "open-url-in-simulator.command",
+      "title": "Open URL in Simulator",
+      "subtitle": "Xcode",
+      "description": "Open the given URL in the currently booted simualtor.",
+      "mode": "no-view",
+      "arguments": [
+        {
+          "name": "URL",
+          "placeholder": "URL to open...",
```

Comment:
I think other commands which are making use of the Arguments API are simply displaying the name 🤔 so maybe removing the dots would be a good idea.

<img width="319" alt="Bildschirm­foto 2023-02-03 um 17 45 25" src="https://user-images.githubusercontent.com/11733014/216659278-1accdca9-ec99-41c7-8c9a-4f4b8da3d09d.png">